homeassistant/components/smartthings/sensor.py

```diff
@@ -331,7 +331,6 @@ class SmartThingsSensorEntityDescription(SensorEntityDescription):
                 translation_key="dryer_machine_state",
                 options=WASHER_OPTIONS,
                 device_class=SensorDeviceClass.ENUM,
-                deprecated=lambda _: "machine_state",
```

Comment:
there are more for this

Reply:
There are not other deprecated machine states